app/controllers/service_controller.rb

```diff
@@ -141,6 +141,15 @@ def service_form_fields
 
   private
 
+  def sanitize_output(stdout)
```

Comment:
```    
    htm = stdout.gsub('"', '\"')
    htm.gsub!("'", "\\\\'")
    htm.gsub!(/{{/, '\{\{')
    htm.gsub!(/}}/, '\}\}')
    htm
```

This is actually a copy of the `gsub` pattern that will work.

The files coming back from Ansible are prefixing `{{` with a `"` which needs to be stripped out of the document before we can `gsub!` on the returned string. Not sure this solution is the most efficient - but it is the exact code running on the appliance that allows the output to show.